### PR TITLE
Add alias to series name. Change existing tests to reflect this

### PR DESCRIPTION
**Pull request by elainearbaugh**

### CODE REVIEW

File: cabot/metricsapp/api/elastic.py [hunk 1]
```diff
@@ -1,7 +1,8 @@
 from django.conf import settings
 from django.core.exceptions import ValidationError
 from elasticsearch import Elasticsearch
-from cabot.metricsapp.defs import ES_SUPPORTED_METRICS, ES_VALIDATION_MSG_PREFIX, HIDDEN_METRIC_SUFFIX
+from cabot.metricsapp.defs import ES_SUPPORTED_METRICS, ES_VALIDATION_MSG_PREFIX, HIDDEN_METRIC_SUFFIX, \
```
Comment (mahendra) [inline cabot/metricsapp/api/elastic.py]: nit: I would recommend

```python
from cabot.metricsapp.defs import (
    ES_SUPPORTED_METRICS,
    ES_VALIDATION_MSG_PREFIX,
    ..,
)
```

Alternatively
```python
from ... import defs

use defs. ES_SUPPORTED_METRICS, defs.HIDDEN_METRIC_SUFFIX etc.
```

File: cabot/metricsapp/api/grafana_elastic.py [hunk 2]
```diff
@@ -95,7 +96,14 @@ def _add_aggs(search_aggs, series, min_time, default_interval):
     pipeline_id_name_mapping = dict()
     pipeline_metrics = []
     for metric in series['metrics']:
-        metric_type = metric_name = metric['type']
+        metric_type = metric['type']
```
Comment (mahendra) [inline cabot/metricsapp/api/grafana_elastic.py]: move this logic to a function?

File: cabot/metricsapp/api/elastic.py [hunk 3]
```diff
@@ -49,6 +50,8 @@ def validate_query(query, msg_prefix=ES_VALIDATION_MSG_PREFIX):
     for metric, items in query.iteritems():
         # If metric is hidden ignore the hidden suffix
         metric = metric.replace('_{}'.format(HIDDEN_METRIC_SUFFIX), '')
+        # Get rid of a possible alias name
+        metric = metric.split(ALIAS_DELIMITER)[0]
```
Comment (mahendra) [inline cabot/metricsapp/api/elastic.py]: move to a function?

File: cabot/metricsapp/api/elastic.py (hunk 1 above)
Comment (elainearbaugh) [inline cabot/metricsapp/api/elastic.py]: done

**Pull request merged**